File: platform/sdk/python/examples/cloud/cloud-002_delete_bucket.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("temp-bucket")
client.deleteBucket("temp-bucket")
print("XJ-OK: cloud-002_delete_bucket")
sys.exit(0)

File: platform/sdk/nodejs/examples/cloud/cloud-002_delete_bucket.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("temp-bucket");
client.deleteBucket("temp-bucket");
console.log("XJ-OK: cloud-002_delete_bucket");
process.exit(0);
